src/main/kotlin/no/nav/syfo/client/aktor/AktorClient.kt

```diff
@@ -23,12 +21,10 @@ class AktorClient(
 ) {
     private val logger = this.logger()
 
-    @Throws(AktørException::class)
```

Comment:
Fjerner `@Throws` fordi de ingen har noen funksjon her. Det er ment for interop med Java-kode.